main.go

```diff
@@ -57,38 +62,36 @@ func main() {
 
 	// Configure localMetrics if it errors log the error but continue
 	if err := metrics.ConfigureMetrics(context.TODO(), *metricsServer); err != nil {
-		fmt.Println(err, "Failed to configure Metrics")
+		log.Error(err, "Failed to configure metrics")
 
 	}
 
 	//reading the aws-account-shredder-credentials secret
 	accessKeyID, secretAccessKey, err := k8sWrapper.GetAWSAccountCredentials(context.TODO(), cli)
 	if err != nil {
-		fmt.Println("ERROR: ", err)
+		log.Error(err, "Failed to read aws-account-shredder-credentials secret")
 	}
 
 	// creating a new AWSclient with the information extracted from the secret file
 	awsClient, err := clientpkg.NewClient(accessKeyID, secretAccessKey, "", "us-east-1")
 	if err != nil {
-		fmt.Println("ERROR:", err)
+		log.Error(err, "Failed to create new AWSclient")
 	}
 	for {
 		// reading the account ID to be cleared
 		accountCRList, err := awsv1alpha1.GetAccountCRsToReset(context.TODO(), cli)
 		if err != nil {
-			fmt.Println("ERROR: ", err)
+			log.Error(err, "Failed to retieve list of accounts to clean")
 		}
 
 		for _, account := range accountCRList {
-			fmt.Println("Now Processing :")
-			fmt.Println("AccountCR:", account.Name)
-			fmt.Println("AccountID:", account.Spec.AwsAccountID)
+			logger := log.WithValues("AccountName", account.Name, "AccountID", account.Spec.AwsAccountID)
```

Comment:
I think this message needs a bit more context as to what's happening

Reply:
That is not a message, I simply made the logger always display the accountCR and accountID so there is no longer a need to output that information beforehand.

Reply:
Okay. Then please add a message at the start of processing each account CR

Reply:
Okay. I still think its helpful to have a log that says "Starting cleanup of Account X"

Reply:
I disagree. All that does is create a log that we will never look at and remove later since the information we want will always accessible in other useful logs.

Reply:
This makes it easy to see when a run started on an Account CR. I can just grep for "starting cleanup of osd-mgmt-aaabbb" and then go from there. Without this line, its hard to parse where one run ends and another begins.

Reply:
I'm +1 for having more context here. For me personally it would be easier to search/find the starting point vs looking through all the logs with the accountcr and id. What's the issue with having both the message and the logger? I don't think it would be an overkill @ArielLima

Reply:
I still don't think it will be that useful but I'll add it